src/hooks/init.js

```diff
@@ -19,9 +19,7 @@ module.exports = async function initHooks(context) {
     globalConfig.set('telemetryDisabled', false)
     console.log('Netlify telemetry has been enabled')
     console.log('You can disable it anytime with the --telemetry-disable flag')
-    track('user_telemetryEnabled', {
-      force: true,
```

Comment:
This is the only place `force` was passed. It's redundant as `track` re-reads `telemetryDisabled`